docs/src/user-guide/configuring/rules.md

```diff
@@ -278,7 +290,7 @@ To disable all inline config comments, use the `noInlineConfig` setting. For exa
 }
 ```
 
-This setting is similar to [--no-inline-config](../command-line-interface#--no-inline-config) CLI option.
+You can also use the [--no-inline-config](../command-line-interface#--no-inline-config) CLI option to disable rule comments, in addition to other in-line configuration.
```

Comment:
```suggestion
You can also use the [--no-inline-config](../command-line-interface#--no-inline-config) CLI option to disable rule comments, in addition to other in-line configurations.
```

Reply:
I believe the singular is correct here but will let @bpmutter comment.

Reply:
Hmm, this one seems tricky. Grammarly shows plural

<img width="764" alt="Screenshot 2022-11-15 at 7 17 35 AM" src="https://user-images.githubusercontent.com/46647141/201806592-9073608d-2087-49a1-9686-dcf2a17b4438.png">

Reply:
I also feel `configuration` to be correct here? as we have only one configuration?

Reply:
`In addition to other` which makes it plural.

In the CLI option, we can pass many configurations, right? eg:

```
eslint index.js --no-eslintrc --env "es6"  --parser-options "{ecmaVersion: 2018}" --no-inline-config
```

Reply:
In this place we were meaning `inline-config`. I guess we specify multiple inline-configs as one inline configuration? (`/* eslint indent: "error", curly: "error" */`)

Reply:
generally the word 'configuration' is only singular - https://www.wordhippo.com/what-is/the-plural-of/configuration.html

'configurations' usually only when there are multiple different *sets* of configurations. 

in this case, i think both could be correct. but singular reads better to me.